components/event-sources/reconciler/object/deployment.go

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright 2019 The Kyma Authors.
```

Comment:
Year 2020

Reply:
As discussed internally: copyright notes are not used in Kyma. Will delete all of them.